definitions/divorce/json/CaseEvent.json

```diff
@@ -2182,7 +2182,7 @@
     "DisplayOrder": 23,
     "PreConditionState(s)": "AwaitingDecreeAbsolute",
     "PostConditionState": "DARequested",
-    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "RetriesTimeoutURLSubmitEvent": "120,120",
```

Comment:
This is not a valid field, and I believe already fixed in https://github.com/hmcts/div-ccd-definitions/pull/260/files#diff-058c7faed859d6ad1b2628fab9c7b226R2187